src/main.rs

```diff
@@ -38,6 +39,37 @@ fn bind_socket(addr: SocketAddr) -> UdpSocket {
     socket
 }
 
+/// Save DHT keys to a binary file.
+fn save_keys(keys_file: String, pk: PublicKey, sk: SecretKey) {
+    let mut file = File::create(keys_file).expect("Failed to create the keys file");
+    file.write(pk.as_ref()).expect("Failed to save public key to the keys file");
```

Comment:
BTW, do we need to save PK? https://docs.rs/sodiumoxide/0.0.16/sodiumoxide/crypto/box_/curve25519xsalsa20poly1305/struct.SecretKey.html#method.public_key

Reply:
No, but it's not something difficult, so it's better to save it as `tox-bootstrapd`. It'll be more convenient for users to migrate to our implementation, and if they find a bug - easier to go back to a c implementation.

Reply:
OK